heartbeat/testutil/hbtestutil.go

```diff
@@ -0,0 +1,79 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package testutil
```

Comment:
In other places we call this package `testing`

Reply:
++ will change

Reply:
both `testing` and `testutil` are somewhat generic. We even have a `testing` package that is not used for testing but for config validation... I like what the go stdlib does. E.g. `httptest` or `iotest`. Depending how generic this package is `hbtest` might be fine :)